libs/component-library/src/lib/theme/inputClasses.ts

```diff
@@ -14,8 +15,8 @@ const defaultProps: TextInputClassType = {
   borderWidth: 1,
   color: 'content1',
   height: 52,
-  justifyContent: 'center',
-  lineHeight: 0,
+  lineHeight: Platform.OS === 'ios' ? 0 : undefined,
+  paddingVertical: '0',
```

Comment:
What is this overriding? Does Text inputs already come with vertical padding?

Reply:
Yes, without this the input value wasn't centering up properly.